activemodel/lib/active_model/serializers/json.rb

```diff
@@ -13,6 +13,92 @@ module JSON
         extend ActiveModel::Naming
 
         class_attribute :include_root_in_json, instance_writer: false, default: false
+        class_attribute :json_key_formatters, instance_writer: false, default: Hash.new { KeyFormatter::Identity.new }
```

Comment:
Ideally, this would be marked with `# :nodoc:`. Does that comment work for attributes, or are there other ways to omit them from sdoc?